csm_web/csm_web/settings.py

```diff
@@ -59,6 +58,13 @@
     "django_filters",
 ]
 
+if DJANGO_ENV == DEVELOPMENT:
+    INSTALLED_APPS.append("django_extensions")
```

Comment:
do you think we might want this in production too? i ended up spending a fair bit of time in shell in production

Reply:
Good point